pkg/apis/storage/util/helpers.go

```diff
@@ -45,6 +45,8 @@ const BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
 //   volume belongs.
 //TODO: Update this to final annotation value as it matches BetaStorageClassAnnotation for now
 const StorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+const StorageClassKubernetesDescription = "kubernetes.io/description"
+const StorageClassDescription = "description"
```

Comment:
I wonder if these definitions and the helpers belong in a more generic place, for future use in other object types...